lib/rpcServer/commands/fetchDocuments.js

```diff
@@ -98,15 +98,15 @@ const fetchDocumentsFactory = (driveAPI) => {
  *                          description: Mongo-like sort field
  *                        limit:
  *                          type: integer
- *                          default: ''
+ *                          default: 1
```

Comment:
I think we need to define max limit and this max limit will be a default value.

Reply:
Yes, I just used 1 as a placeholder since I didn't see any defined limits. Technically `default` is not a required field, but it's convenient to have for auto-generated API sites so the user doesn't have to enter a value manually.

What value should I change this to?

Reply:
If we don't know what this should be yet, I suggest just merging with a reasonable value for now and revisiting in the future when a limit is defined.

Reply:
Let's figure it :). I was thinking about 100.

Reply:
Okay, I can set to 100. Is that your final answer? :slightly_smiling_face:

Reply:
Yep!

Reply:
Done! :+1: